source/admin_manual/list_of_events.rst

```diff
@@ -165,6 +165,247 @@ Global Fields
+---------------------------------
+
++--------------------+-----------------------------------------------------------+
+| Field             | Description                                                |
++===================+============================================================+
+| user              | User to lookup                                             |
++-------------------+------------------------------------------------------------+
+| service           | Service name, such as imap3, pop3, lmtp                    |
++-------------------+------------------------------------------------------------+
+| session           | Session identifier                                         |
++-------------------+------------------------------------------------------------+
+| local_name        | TLS SNI                                                    |
++-------------------+------------------------------------------------------------+
+| local_ip          | Local IP client connected to                               |
++-------------------+------------------------------------------------------------+
+| remote_ip         | Remote IP of client                                        |
++-------------------+------------------------------------------------------------+
+| local_port        | Local port client connected to                             |
++-------------------+------------------------------------------------------------+
+| remote_port       | Remote port of client                                      |
++-------------------+------------------------------------------------------------+
+| real_local_ip     | Real local IP as seen by the server                        |
++-------------------+------------------------------------------------------------+
+| real_remote_ip    | Real remote IP as seen by the server                       |
++-------------------+------------------------------------------------------------+
+| real_local_port   | Real local port as seen by the server                      |
++-------------------+------------------------------------------------------------+
+| real_remote_port  | Real remote port as seen by the server                     |
++-------------------+------------------------------------------------------------+
+
+auth_client_passdb_lookup_finished
+----------------------------------
+
++--------------------+-----------------------------------------------------------+
+| Field             | Description                                                |
++===================+============================================================+
+| user              | User to lookup                                             |
++-------------------+------------------------------------------------------------+
+| service           | Service name, such as imap3, pop3, lmtp                    |
++-------------------+------------------------------------------------------------+
```

Comment:
These should be under "common fields" like done elsewhere to avoid duplication.

Reply:
I have to think. There are two events where these fields are not.